docs/LibraryEvolution.rst

```diff
@@ -903,11 +904,15 @@ not handle new cases well.
 Protocols
 ~~~~~~~~~
 
-There are very few safe changes to make to protocols:
+There are very few safe changes to make to protocols and their members:
 
 - A new non-type requirement may be added to a protocol, as long as it has an
   unconstrained default implementation.
 - A new associated type may be added to a protocol, as long as it has a default.
+  As with any other declarations, newly-added associated types must be marked
+  with ``@available`` specifying when they were introduced.
+- A default may be added to an associated type. Removing a default is a
```

Comment:
Oops, good point. Is it worth allowing the removal if the associated type has been around as long as the protocol, or should we just leave it out?